content/en/docs/reference/setup-tools/kubeadm/kubeadm-upgrade.md

```diff
@@ -33,6 +33,9 @@ Please check these documents out for more detailed how-to-upgrade guidance:
 ## kubeadm upgrade apply  {#cmd-upgrade-apply}
 {{< include "generated/kubeadm_upgrade_apply.md" >}}
 
+## kubeadm upgrade diff {#cmd-upgrade-diff}
+{{< include "generated/kubeadm_upgrade_diff.md" >}}
```

Comment:
Em .... this is important. Even if the reference docs get regenerated, we will still need to manually edit this file.

Reply:
@tengqm can you clarify please? We should not be manually editing any generated docs files.

Reply:
@Bradamant3  The file in question (content/en/docs/reference/setup-tools/kubeadm/kubeadm-upgrade.md) is not generated from tools, the referenced file (e.g. "generated/kubeadm_upgrade_diff.md") is.